include/pmacc/Environment.hpp

```diff
@@ -527,17 +529,16 @@ namespace detail
             if (rc == cuplaSuccess)
             {
 #if (PMACC_CUDA_ENABLED == 1)
-                cudaDeviceProp dprop;
-                CUDA_CHECK((cuplaError_t)cudaGetDeviceProperties(&dprop, tryDeviceId));
-                log<ggLog::CUDA_RT> ("Set device to %1%: %2%") % tryDeviceId % dprop.name;
+                CUDA_CHECK((cuplaError_t)ALPAKA_API_PREFIX(GetDeviceProperties)(&devProp, tryDeviceId));
+                log<ggLog::CUDA_RT> ("Set device to %1%: %2%") % tryDeviceId % devProp.name;
```

Comment:
Same as previous comment, should this stay CUDA-only?

Reply:
fixed by activating this code for HIP too